builder/builder-next/adapters/snapshot/snapshot.go

```diff
@@ -74,6 +74,10 @@ func NewSnapshotter(opt Opt) (snapshot.SnapshotterBase, error) {
 	return s, nil
 }
 
+func (s *snapshotter) Name() string {
+	return "default"
```

Comment:
Wasn't entirely sure where the name is used, and what to name this one, so let me know what I should use here

Reply:
@tonistiigi WDYT? `moby` seems more consistent?

Reply:
It is not used anywhere in moby so doesn't matter much. I'm fine with either.